functional_tests/functional_test.go

```diff
@@ -1031,6 +1031,25 @@ var _ = ginkgo.Describe("Tests `appgw.ConfigBuilder`", func() {
 			check(cbCtx, "waf_annotation.json", stopChannel, ctxt, configBuilder)
 		})
 
+		ginkgo.It("WAF Priority Annotation", func() {
+			annotatedIngress := newIngressSlashNothingSlashSomething()
+			annotatedIngress.Annotations[annotations.RequestRoutingRulePriority] = "100"
+
+			cbCtx := &ConfigBuilderContext{
+				IngressList: []*networking.Ingress{
+					annotatedIngress,
```

Comment:
Can you add a 2nd simple ingress in this list and show an example of how it's priority will look n comparison to the annotated ingress ?

Reply:
If I add a second ingress I should also add the priority annotation to it. What is the goal?

Reply:
I want to see that other ingress continues to be assigned generated priorities.